shared/middleware/helpers/storage_id.rb

```diff
@@ -8,7 +8,7 @@ def create_storage_id_cookie
   response.set_cookie(storage_id_cookie_name, {
     value:CGI.escape(storage_encrypt_id(storage_id)),
     domain:".#{site}",
-    path:'/v3/apps',
+    path:'/',
```

Comment:
Danger! For this to work behind varnish, we would need to pass the "storage" cookie to "/editcode/5" and to any "share link" (which for applab will be changing soon to use the new "project share links"). This may not be not feasible, but presenting here to illustrate what I'm trying to accomplish.